### PR TITLE
small changes - remove useless code

### PR DESCRIPTION
**Pull request by vince62s**

### CODE REVIEW

File: onmt/modules/conv_multi_step_attention.py [hunk 1]
```diff
@@ -46,22 +45,12 @@ def forward(self, base_target_emb, input_from_dec, encoder_out_top,
                 which is the combination of base emb and top output of encode
         """
 
-        # checks
-        # batch, channel, height, width = base_target_emb.size()
```
Comment (francoishernandez) [inline onmt/modules/conv_multi_step_attention.py:50]: Maybe document the dimensions of `base_target_emb` and other tensors in the docstring, just to keep it somewhere explicitly.

File: onmt/modules/global_attention.py [hunk 2]
```diff
@@ -108,9 +108,6 @@ def score(self, h_t, h_s):
         # Check input sizes
```
Comment (francoishernandez) [inline onmt/modules/global_attention.py:108]: We don't check much anymore...

File: onmt/modules/global_attention.py [hunk 3]
```diff
@@ -205,23 +197,13 @@ def forward(self, source, memory_bank, memory_lengths=None, coverage=None):
...
-            aeq(batch, batch_)
-            aeq(dim, dim_)
             batch_, source_l_ = align_vectors.size()
-            aeq(batch, batch_)
-            aeq(source_l, source_l_)
 
         else:
             attn_h = attn_h.transpose(0, 1).contiguous()
             align_vectors = align_vectors.transpose(0, 1).contiguous()
             # Check output sizes
```
Comment (francoishernandez) [inline onmt/modules/global_attention.py:205]: Again.

File: onmt/trainer.py [hunk 4]
```diff
@@ -260,13 +259,6 @@ def train(self,
             # UPDATE DROPOUT
             self._maybe_update_dropout(step)
 
-            if self.gpu_verbose_level > 1:
-                logger.info("GpuRank %d: index: %d", self.gpu_rank, i)
-            if self.gpu_verbose_level > 0:
-                logger.info("GpuRank %d: reduce_counter: %d \
-                            n_minibatch %d"
-                            % (self.gpu_rank, i + 1, len(batches)))
```
Comment (francoishernandez) [inline onmt/trainer.py:268]: Are we 100% confident this can't be useful anymore in some debugging scenario?

Comment (vince62s) [reply onmt/trainer.py:268]: let's remove it for now, I am keeping the option in and when I will re-address the multi-node we'll check again.

**Pull request merged**